### PR TITLE
Added CentOS/RH dependencies for python crypto

### PR DESCRIPTION
**Pull request by punkrokk**
From https://cryptography.io/en/latest/installation/#building-cryptography-on-linux
sudo yum install gcc libffi-devel python-devel openssl-devel

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ This pack consists of a sample JIRA sensor and a JIRA action.
 ## Installation
 
 You will need to have `gcc` installed on your system. For Ubuntu systems, run `sudo apt-get install gcc`. For Redhat/CentOS
-systems, run `sudo yum install gcc`.
+systems, run `sudo yum install gcc libffi-devel python-devel openssl-devel`. To build the python cryptography dependency (part of the following "st2 pack install command" command, 2GB of RAM is recommended. In some cases adding a swap file may elimiate strange gcc compiler errors.
```
Comment (emedvedev) [inline README.md]: A closing bracket is lost somewhere :)

**Comment by emedvedev** [comment]
Thanks for the fix! There's a small typo, other than that everything's fine :)

File: README.md (hunk 1 above)
Comment (LindsayHill) [reply README.md]: And also s/elimiate/eliminate/

**Comment by punkrokk** [comment]
fixed

**Comment by emedvedev** [comment]
Still missing the closing bracket :)

**Comment by punkrokk** [comment]
really fixed now

**Comment by emedvedev** [comment]
Thanks :)

**Pull request merged**